src/libespeak-ng/CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ target_link_libraries(espeak-ng PRIVATE espeak-ng-config ucd)
 if (NOT MSVC)
   target_link_libraries(espeak-ng PRIVATE m)
 endif()
-target_link_libraries(espeak-ng PUBLIC espeak-include)
```

Comment:
This line using `PUBLIC` causes errors when building for linux aarch64. I have spent half a day to figure it out.

https://github.com/rhasspy/espeak-ng/tree/master/src/include/compat
The above folder contains many header files that share the same name with the standard library headers. It is really bad to also include it in the header search path.

(It is even worse to name them using standard library header filenames.)